#3.3.3

nayansarkar opened the pull request:
* feature: SNSService.sendToTopic --> Added the ability to set message attributes so that messages can be filtered by attribute(s). To receive only a subset
 of the messages, a subscriber will need to assign filter policy to the topic subscription.

nayansarkar committed:
#3.3.3
* feature: SNSService.sendToTopic --> Added the ability to set message attributes so that messages can be filtered by attribute(s). To receive only a subset
 of the messages, a subscriber will need to assign filter policy to the topic subscription.

andrewcampkin reviewed (changes requested):
Please add an explanation and example to the read me.

nayansarkar committed:
#3.3.3
* feature: SNSService.sendToTopic --> Added the ability to set message attributes so that messages can be filtered by attribute(s). To receive only a subset
 of the messages, a subscriber will need to assign filter policy to the topic subscription. Added unit test
